travis/cfg/travis_run_pylint_pr.cfg

```diff
@@ -3,7 +3,7 @@ profile=no
 ignore=CVS,.git,scenarios,.bzr
 persistent=yes
 cache-size=500
-load-plugins=pylint.extensions.check_docstring,pylint.extensions.check_mccabe
+load-plugins=pylint.extensions.docstyle, pylint.extensions.mccabe
```

Comment:
I thought that but if you don't make the change, an error appears into
tests like module check_docstring not found

El ago. 2, 2016 8:42 PM, "moylop260" <notifications@github.com> escribió:

> In travis/cfg/travis_run_pylint_pr.cfg
> <https://github.com/Vauxoo/maintainer-quality-tools/pull/131#discussion_r73268605>
> :
>
> > @@ -3,7 +3,7 @@ profile=no
> >  ignore=CVS,.git,scenarios,.bzr
> >  persistent=yes
> >  cache-size=500
> > -load-plugins=pylint.extensions.check_docstring,pylint.extensions.check_mccabe
> > +load-plugins=pylint.extensions.docstyle, pylint.extensions.mccabe
>
> This file is overwrite by the wget of pylint-conf.
> This change is unnecessary.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Vauxoo/maintainer-quality-tools/pull/131/files/8220900964d67c349038b193cd32b20fcbb43234#r73268605>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ALMoyHXhuJVFLdHywklh0sRlxJz_Ofzbks5qb_HdgaJpZM4Ja516>
> .
>